boards/acd52832/board.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_acd52832
+ * @{
+ *
+ * @file
+ * @brief       Board initialization for the nRF52 DK
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
```

Comment:
author and copyright should  be you? Also the brief description is wrong for this board.